lib/assessment.rb

```diff
@@ -20,21 +20,21 @@ def sum(enum)
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    odd?
```

Comment:
Yes, you will want to use `odd?`, but you want to use that method to evaluate each element in `enum`. Are there any enumerable methods that we might use to check if at least some elements in our collection pass a given test?